build_cmake/scripts/check_deps.py

```diff
@@ -123,8 +123,9 @@ def main(dir: str, branch: str):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Validate submodules for PR')
-    parser.add_argument('branch', type=str, help="The branch to check for in non-couchbasedep repos")
+    parser.add_argument('-branch', type=str, help="The branch to check for in non-couchbasedep repos")
+    parser.add_argument('-path', type=str, help="The path to the parent repo to check")
```

Comment:
I'd rather have these both as positional arguments, or at the very least have them both marked as `required=True` because now as it stands you can omit either one as a valid invocation (`-` and `--` arguments are considered optional)

Reply:
Will make it as positional arg